config/samples.tsv

```diff
@@ -1,5 +1,5 @@
 sample	condition	batch_effect
-A	treated	batch1
-B	untreated	batch1
-C	treated	batch2
-D	untreated	batch2
+A   treated batch1
+B   untreated   batch1
+C   treated batch2
+D   untreated   batch2
```

Comment:
Please restore the old sheet (after your changes, it contains spaces instead of tabs, but this is a .tsv file).

Reply:
Done.